docs/4.x/security-in-piwik.md

```diff
@@ -66,6 +66,12 @@ Or using dependency injection by creating a `plugins/MyPlugin/config/config.php`
 return array(
     \Piwik\View\SecurityPolicy::class => DI\decorate(function ($previous) {
         /** @var \Piwik\View\SecurityPolicy $previous */
+        
+        if (!\Piwik\SettingsPiwik::isMatomoInstalled()) {
```

Comment:
@AltamashShaikh that would be needed though only if a system setting is accessed as part of this? I suppose it might not hurt to have it in the example though.

Reply:
Yes, removed the comment as discussed